lib/sensu/transport/rabbitmq.rb

```diff
@@ -185,18 +186,34 @@ def next_connection_options
 
       def setup_connection(options={})
         reconnect_callback = Proc.new { reconnect }
+        on_possible_auth_failure = Proc.new {
+          @logger.warn("transport connection error", :reason => "possible authentication failure. wrong credentials?")
```

Comment:
We already debug log authentication information in core https://github.com/sensu/sensu/blob/master/lib/sensu/daemon.rb#L248-L251